packages/database/lib/migrations/20240611074119_update_external_webhooks_nullable.cjs

```diff
@@ -0,0 +1,16 @@
+const TABLE = '_nango_external_webhooks';
+
+exports.up = function (knex) {
+    return knex.schema.alterTable(TABLE, function (t) {
+        t.string('primary_url').nullable().alter();
+        t.string('secondary_url').nullable().alter();
+        t.unique(['environment_id']);
+    });
+};
+
+exports.down = function (knex) {
+    return knex.schema.alterTable(TABLE, function (t) {
+        t.string('primary_url').notNullable().alter();
+        t.string('secondary_url').notNullable().alter();
+    });
```

Comment:
I don't believe in Down, but you need to remove unique index

Reply:
Updated